frontend/src/useDocumentScrollThrottled.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react'
+import { throttle } from 'lodash'
+
+export default function useDocumentScrollThrottled(callback) {
+  const [, setScrollPosition] = useState(0)
```

Comment:
Was tut das KOmma in Zeile 5 in den []?

Reply:
ich brauch das erste Argument nicht, dann kann man es so weglassen.